mne/preprocessing/__init__.py

```diff
@@ -21,3 +21,4 @@
 from .stim import fix_stim_artifact
 from .maxwell import maxwell_filter
 from .xdawn import Xdawn
+from .optical_density import optical_density
```

Comment:
to avoid sometimes annoying module / func name collisions it would be better to name your file `_optical_density.py`. That way `mne.preprocessing._optical_density` is clearly the module and `mne.preprocessing.optical_density` the function.